### PR TITLE
fixed a white page problem when loading and continuing to implement data from the API

### PR DESCRIPTION
**Pull request by piotrpaczuski**

### CODE REVIEW

**Comment by piotrpaczuski** [comment]
All data is retrieved from the API, take a look at my code.

File: src/layouts/PersonDetails/index.js [hunk 1]
```diff
@@ -7,41 +7,84 @@ import {
...
       <ContentBackground>
         <Content>
           <TileContent>
-            <Tile personDetails={"personDetails"} srcImage={person} />
+            <Tile personDetails profileSizes={profileSizes} />
           </TileContent>
           <CastContent>
-            <SubdivTitle>Movies - cast()</SubdivTitle>
+            <SubdivTitle>
+              Movies - cast({peopleCredits.cast && peopleCredits.cast.length})
```
Comment (KonradBauer) [inline src/layouts/PersonDetails/index.js:30]: People credits going out of tile

**Review by KonradBauer — APPROVED** [review]
I checkes People and PersonDetails and all data are downloaded corectly from API. In PersonDetails the person description going overlay from tile, but it's probably tile thing.

**Comment by piotrpaczuski** [comment]
Yes, person description who going overlay from tile is tile thing, I will fix it tommorow.

File: src/layouts/PersonDetails/index.js (hunk 1 above)
Comment (piotrpaczuski) [reply src/layouts/PersonDetails/index.js:30]: What do you have on mind?

Comment (KonradBauer) [reply src/layouts/PersonDetails/index.js:30]: I indicate in which component the description goes beyond the tile.

Comment (piotrpaczuski) [reply src/layouts/PersonDetails/index.js:30]: You have on mind mobile view?

Comment (KonradBauer) [reply src/layouts/PersonDetails/index.js:30]: that's right, it was mobile view

Comment (piotrpaczuski) [reply src/layouts/PersonDetails/index.js:30]: OK, I will fix it tommorow.

**Review by florekdeveloper — APPROVED** [review]
Well done, mobile view looks good.

**Pull request merged**